cmd/capabilities/dispatcher/dispatcher.go

```diff
@@ -0,0 +1,169 @@
+// Sample run-helloworld is a minimal Cloud Run service.
+package main
+
+import (
+        "log"
+        "time"
+        "os"
+        "context"
+        "runtime"
+        "encoding/json"
+        "cloud.google.com/go/pubsub"
+        "github.com/apigee/registry/rpc"
+	    "github.com/golang/protobuf/jsonpb"
+        "google.golang.org/grpc/codes"
+        "google.golang.org/grpc/status"
+        "github.com/apigee/registry/cmd/capabilities/utils"
+        cloudtasks "cloud.google.com/go/cloudtasks/apiv2"
+        taskspb "google.golang.org/genproto/googleapis/cloud/tasks/v2"
+)
+
+// Make this configurable
```

Comment:
This looks like it should be marked TODO

Reply:
Done. This is a clean-up or fixit kind of task. Will track this in a separate issue.

Reply:
I will track these cleanup/fixit efforts in a different issue. And yes, these are not essential for the dispatcher to function correctly.